### DISCUSSION TITLE
1.19.2?

### DISCUSSION THREAD

**Discussion by DragonMaster14545** [Sinytra/ForgifiedFabricAPI#20] [Q&A] [answered]
I am sorry to be that guy but since there is no discord, I have to ask here: Is there any chance that this will get a port to 1.19.2? Because most of the mods are on that version right now. Thanks in advance

**Comment by Su5eD** [comment, marked as answer]
Hi, there are currently no plans for a backport due to the maintenance costs associated with it. I recognize that 1.19.2 has become a popular version, but our focus right now is on future versions, which will hopefully let us simplify the code of FFAPI.

**Reply by DragonMaster14545** [reply]
Ok thanks a lot for that info.

**Reply by lkhklij** [reply]
you what's better then 1.19.2, _1.18.2_
